### ISSUE TITLE
md-tab label not visible

### ISSUE THREAD

**Issue by groupsvkg** [angular/material#5402]
md-tab label text is not visible.
```html
<md-content class="md-whiteframe-5dp" layout-margin>
		    <md-tabs md-dynamic-height md-border-bottom>
		    	<md-tab label="one">
			        <md-content class="md-padding">
			          <h1 class="md-display-2">Tab One</h1>
			          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla venenatis ante augue. Phasellus volutpat neque ac dui mattis vulputate. Etiam consequat aliquam cursus. In sodales pretium ultrices. Maecenas lectus est, sollicitudin consectetur felis nec, feugiat ultricies mi.</p>
			        </md-content>
		      	</md-tab>
		      	<md-tab label="two">
			        <md-content class="md-padding">
			          <h1 class="md-display-2">Tab Two</h1>
			          <p>Lorem ipsum .</p>
			        </md-content>
		      	</md-tab>
		    </md-tabs>
</md-content>
```
bower.json dependencies
```json
"dependencies": {
    "angular-material": "~0.11.4",
    "angular-ui-router": "~0.2.15",
    "angular-cookies": "~1.4.7",
    "angular-resource": "~1.4.7",
    "angular-sanitize": "~1.4.7",
    "angular-messages": "~1.4.7",
    "es5-shim": "~4.1.15",
    "json3": "~3.3.2"
  }
```
![image](https://cloud.githubusercontent.com/assets/366335/10779970/12287222-7d5e-11e5-9eff-1f0c21dc1740.png)

**Comment by reppners** [comment]
Same here. Label styling is broken with the default theme.

**Comment by groupsvkg** [comment]
i used following css and its ok for me

md-tab-item span{
  color: rgb(255,128,171);
}

**Comment by robertmesserle** [comment]
I believe this was a temporary bug in `1.0-rc1` and was immediately fixed in `1.0-rc2` - please confirm?  (Closing for now, will reopen)

**Comment by Vipul-Antino** [comment]
I have encountered this bug too
